### ISSUE TITLE
[Accessibility]: Make "Go to line" announce focused line after executed

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#179283]
CC @meganrogge

This is not a regression.

If you use "Go to line (`Ctrl+G`)" and land
in a specific line, screen reader does not read the current line b/c there's no announcement.

You may recall this similar issue that you encountered when addressing Alt+F5 (go to the next diff) did not announce the focused line and you used a selection to fire an announcement for screen readers.

I have found that `editor.action.wordHighlight.next` (F7) auto-announce the next focus when this key is pressed. This command may be worth looking into. I am not sure whether it also used the selection though.
